### DISCUSSION TITLE
Vercel Project IP

### DISCUSSION THREAD

**Discussion by yukisnowflake** [vercel/community#2562] [Help] [answered]
### Summary

How to get IP address or IP sub-network for allow access to request another resource?

**Comment by amyegan** [comment, marked as answer]
Vercel deployments use dynamic IP addresses because of the nature of the platform. This means it is not possible to predict a single deployment IP address or range because it may change at any time as the deployment scales across instances and regions.

You can learn more about this, and an alternative solution, here: https://vercel.com/guides/how-to-allowlist-deployment-ip-address
